File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseCreateFile.java
package app.documents.core.network.manager.models.response;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import app.editors.manager.mvp.models.base.Base;
import app.editors.manager.mvp.models.explorer.CloudFile;

public class ResponseCreateFile extends Base {

    @SerializedName(KEY_RESPONSE)
    @Expose
    private CloudFile response;

    public CloudFile getResponse() {
        return response;
    }

    public void setResponse(CloudFile response) {
        this.response = response;
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseCreateFile.kt
package app.documents.core.network.manager.models.response

import app.documents.core.network.common.models.BaseResponse
import app.documents.core.network.manager.models.explorer.CloudFile
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ResponseCreateFile(
    @SerializedName(KEY_RESPONSE)
    @Expose
    var response: CloudFile? = null
) : BaseResponse()